From 9694440cd501a9b170fec7c85fe57f1e4fe749b3 Mon Sep 17 00:00:00 2001
From: Keith Packard <keithp@keithp.com>
Date: Thu, 26 Sep 2024 08:23:48 -0700
Subject: [PATCH] src: Add more includes to ao.h

This makes sure altos has all of the C library definitions it needs
rather than on relying on the C library to expose more than POSIX
says it should.

Signed-off-by: Keith Packard <keithp@keithp.com>
---
 src/kernel/ao.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/kernel/ao.h b/src/kernel/ao.h
index b3f780f5c..0437cb07f 100644
--- a/src/kernel/ao.h
+++ b/src/kernel/ao.h
@@ -19,10 +19,11 @@
 #ifndef _AO_H_
 #define _AO_H_
 
-#include <stdint.h>
+#include <inttypes.h>
 #include <stdio.h>
 #include <string.h>
 #include <stddef.h>
+#include <stdarg.h>
 #include <stdbool.h>
 #include <ao_pins.h>
 #include <ao_arch.h>
-- 
2.45.2

